jboss-kie-smartrouter/added/launch/jboss-kie-smartrouter.sh

```diff
@@ -47,6 +48,25 @@ function configure_mem_ratio() {
     export JAVA_INITIAL_MEM_RATIO=${JAVA_INITIAL_MEM_RATIO:-25}
 }
 
+function configure_logger_config_file() {
+  # use a default level if the env var is unset
+  if [ -z ${LOG_LEVEL} ]; then
+	  LOG_LEVEL=${LOG_LEVEL:-INFO}
+  fi
+  sed -i "s/{LOG_LEVEL}/${LOG_LEVEL}/" ${CONFIG_DIR}/logging.properties
+  # write all packages with loglevel
+
+  PACKAGES_LOG_LEVEL=$(echo $PACKAGES_LOG_LEVEL_CONF | awk '{gsub(",", "\\n")}1')
+
+  if [ -z "${PACKAGES_LOG_LEVEL_CONF}" ]; then
```

Comment:
I don't think you need to do this.
if the env is not set, the replacement will be for nothing anyways.